spellcheck/platform/linux/spellcheck_linux.cpp

```diff
@@ -146,7 +166,8 @@ void Init() {
 }
 
 bool IsAvailable() {
-	return EnchantSpellChecker::instance()->isAvailable();
+	static auto Available = enchant::loader::do_explicit_linking();
+	return Available;
```

Comment:
@ilya-fedin I think I can accept these changes if you put them on a separate commit. (WIth removing of the `EnchantSpellChecker::isAvailable` method.)